### PR TITLE
Added Quota calculations for reconfigure resize disks

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Added test for disk_resize

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1533263

@miq-bot add_reviewer @tinaafitz 
@miq-bot add_label enhancement

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@tinaafitz Please Review

File: content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb [hunk 1]
```diff
@@ -146,6 +146,11 @@ def vmdb_object(model, id)
   $evm.vmdb(model, id.to_i) if model && id
 end
 
+def get_disk_size(disk_name)
+  mydisk = $evm.vmdb('disk').find_by(:filename => disk_name)
```
Comment (mkanoor) [inline content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb]: @billfitzgerald0120 
Should we log disk_names that are not found and raise an error?
Also add a test case where the disk_name doesn't exist.

Comment (billfitzgerald0120) [reply content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb]: @mkanoor Made changes as requested.

File: content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb [hunk 2]
```diff
@@ -146,6 +146,12 @@ def vmdb_object(model, id)
...
+  raise "ERROR - Disk_name not found for Reconfiguration" if mydisk.nil?
```
[the remark this answers is not in the document]
Comment (billfitzgerald0120) [reply content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb]: @mkanoor Included the disk name when we raise the exception, Please review

**Pull request closed without merging**